ui/parameterlistmodel.cpp

```diff
@@ -62,20 +62,40 @@ void ParameterListModel::updateAlgParameters(QString algName) {
 void ParameterListModel::createCommand(QString algName) {
```

Comment:
Given how you've updated this algorithm (i.e., how it does the instantiating now as opposed to just creating a string that the ScriptEngine processes), you may want to consider renaming it and updating its documentation in the `.h` file.